.changeset/chilly-waves-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/keystone': patch
+---
+
+Added a function `createGraphQLSchema` to `createSystem` to isolate schema generation code.
```

Comment:
tbh I wouldn't ship a changeset for this kind of internal refactor, unless the new function is being exported and the API or functionality of the package changes. Could be wrong, just don't see the reasoning.